js/maps-setup.js

```diff
@@ -55,26 +55,40 @@ function initializeMap() {
                                  my_map_options);
     // this is an *array* that holds all the marker info
     var all_my_markers =
-            [{position: new google.maps.LatLng(41.9000,12.5000),
+            [{position: new google.maps.LatLng(49.339807, -0.584757),
               map: my_map,
               icon: blueURL, // this sets the image that represents the marker in the map to the one
                              // located at the URL which is given by the variable blueURL, see above
-              title: "first Marker",
-              window_content: "<h1>Marker1</h1><p> and this would be the extended description</p>"
+              title: "Gold Beach",
+              window_content: "<h2>Gold Beach</h2><p>Gold Beach (Gold being short for Gold Fish) was one of two beachheads assigned to the British during D-Day.</p>"
              },
-             {position: new google.maps.LatLng(41.8902,12.4923),
+             {position: new google.maps.LatLng(49.310013, -0.328623),
               map: my_map,
               icon: blueURL, // this sets the image that represents the marker in the map
-              title: "second Marker",
-              window_content: "<h1>Marker2</h1><p> and <a href='http://something'>this would</a> be the extended description</p>"
+              title: "Sword Beach",
+              window_content: "<h2>Sword Beach</h2><p>Sword Beach (Short for Sword Fish) was the second beach assigned to British during D-Day.</p>"
             },
-            {position: new google.maps.LatLng(41.8986,12.4768),
+            {position: new google.maps.LatLng(49.335750, -0.452584),
              map: my_map,
              icon: redURL, // this sets the image that represents the marker in the map
```

Comment:
Might be good to comment on why this one is in red (presumably b/c it's the Canadian beach).